CMakeLists.txt

```diff
@@ -656,8 +656,6 @@ if(EXECUTORCH_BUILD_PYBIND)
       etdump
       executorch
       extension_data_loader
-      portable_ops_lib
```

Comment:
Is this added as a transitive dep somewhere, or are you removing the portable ops altogether?